docs/about/internals/releases.md

```diff
@@ -4,94 +4,124 @@ sidebar_position: 6
-ping the next person to pass the baton.
+- For everyone: give project a visible "pulse".
+- For TigerBeetle developers: tell fine grained project evolution story, form
+  shared context, provide material for the monthly newsletter.
+- For TigerBeetle users: inform about all visible and potentially relevant
+  changes.
 
-Before triggering a release, update CHANGELOG.md with the changes since the last release. Run the
-following command to create a release branch and generate changelog skeleton:
+As such:
 
-```console
-$ ./zig/zig build scripts -- changelog
-```
+- Consider skipping over trivial changes in the changelog.
+- Don't skip over meaningful changes of the internals, even if they are not
+  externally visible.
+- If there is a story behind a series of pull requests, tell it.
+- And don't forget the TigerTrack of the week!
 
-After the changelog PR is merged, trigger a release  by pushing a commit from `origin/main` to
-`origin/release`:
+## Release Logistics
 
-```console
-$ git fetch origin && git push origin origin/main:release
-```
+Releases are triggered manually, on Monday. Default release rotation is on the
+devhub: <https://tigerbeetle.github.io/tigerbeetle/>.
 
-This triggers the `release.yml` workflow. The workflow doesn't run unit-tests,
-[not rocket science](https://graydon2.dreamwidth.org/1597.html) rule is assumed. The workflow
-requires an approval of at least one other person.
+The middle name is the default release manager for the _current_ week. They should execute [Release
+Manager Algorithm](#release-manager-algorithm) on Monday. If the release manager isn't available on
+Monday, a volunteer picks up that release.
 
-If the release manager isn't available on Monday, a volunteer picks up that release. Releases can be
-skipped if there are not enough people to make a release (at least two are required) or if there
-aren't any changes in a given week.
+## Skipping Release
 
-## Changelog Guidelines
+Because releases a frequent, it is not a problem to skip a single release. In fact, allowing to
+easily skip a release is one of the explicit purposes of the present process.
 
-Purposes of the changelog:
+If there's any pull request that we feel pressured should land in the next release, the default
+response is to land the PR under its natural pace, and skip the release instead.
 
-- For everyone: give project a visible "pulse".
-- For TigerBeetle developers: tell fine grained project evolution story, form shared context,
-  provide material for the monthly newsletter.
-- For TigerBeetle users: inform about all visible and potentially relevant changes.
+Similarly, if there's a question of whether we should do a release or to skip one, the default
+answer is to skip. Skipping is cheap!
 
-As such:
+If the release is skipped, the changelog is still written and Merged on monday, using the following
+header: `## TigerBeetle - (unreleased)`.
```

Comment:
Can we use (or drop) the _dash_ `-` whether we have a version number or `(unreleased)`?